### ISSUE TITLE
[bug] megaport_gcp_connection does not setup BGP

### ISSUE THREAD

**Issue by mi1os** [megaport/terraform-provider-megaport#21]
I have following tf code that I would expect to create MCR + VXC to google. However when this is applied the VXC is created but no BGP is configured. When I add such VXC manually via UI it works as expected.

```
data "megaport_location" "par_eq2" {
  name = "Equinix PA2/3"
}
data "megaport_location" "par_ix5" {
  name = "Interxion PAR5"
}

resource "megaport_mcr" "sbx_par_1" {
  mcr_name    = "sbx-par-1"
  location_id = data.megaport_location.par_eq2.id

  router {
    port_speed    = 1000
    requested_asn = var.megaport_asn
  }
}

data "megaport_partner_port" "gcp_port_par_fra_1" {
  connect_type = "GOOGLE"
  company_name = "Google Inc"
  product_name = "Paris (cdg-zone1-1536)"
  location_id  = data.megaport_location.par_ix5.id
}

resource "megaport_gcp_connection" "sbx_par_1_gcp_fra_1" {
  # TODO https://github.com/megaport/terraform-provider-megaport/issues/3
  #  admin_locked = true

  vxc_name   = "sbx-par-1-gcp-fra-1"
  rate_limit = 200

  a_end {
    port_id = megaport_mcr.sbx_par_1.id
  }
  csp_settings {
    pairing_key = "XXX"
    requested_product_id = data.megaport_partner_port.gcp_port_par_fra_1.id
  }
}
```

**Comment by pushee** [comment]
Hi @mi1os 
A fix for this issue was just released under v0.2.2, can you please see if using this version resolves this issue?

**Comment by pushee** [comment]
I'll close this issue as we believe it has been resolved and I have not heard back from you - please don't hesitate to reach out if you find this is still an issue.
